spec/mailers/reminders_mailer_spec.rb

```diff
@@ -4,7 +4,12 @@
   let!(:test_event) { FactoryGirl.create(:event, id: 10, name: "Event Test", description: "Great", date: "2019-04-14 20:22:21",
      location: "Here", tickets: 10, capacity: 10) }
   let!(:test_guest) { FactoryGirl.create(:guest, event_id: test_event.id, firstname: "Andy", lastname: "P", email: "a@gmail.com") }
-  describe "remind" do
+  let!(:test_event) { FactoryGirl.create(:event, id: 1) }
+  let!(:test_volunteer) { FactoryGirl.create(:test_volunteer, id: 1)}
+  let!(:test_task) { FactoryGirl.create(:task, event_id: 1, id: 1)}
```

Comment:
try to only put let statements in the block that they're necessary in. I just refactored our tests on my branch, maybe look at that to get some ideas.